src/components/Composer/index.js

```diff
@@ -119,22 +109,24 @@ class Composer extends React.Component {
     constructor(props) {
         super(props);
 
-        const initialValue = props.defaultValue
+        this.initialValue = props.defaultValue
             ? `${props.defaultValue}`
             : `${props.value || ''}`;
 
+        this.selection = {
+            start: this.initialValue.length,
+            end: this.initialValue.length,
+        };
```

Comment:
yeah i see, i removed the `this.selection` and `this.initialValue` fields. However there seems to be a bug with RN web, where using an input with `defaultValue` set the selection cursor at the start instead of the end. I added a comment to the line where I manually set the selection.
In the future it would be cool to resolve this bug ofc